conf/supervisord.conf

```diff
@@ -12,21 +12,23 @@ stdout_events_enabled=true
 stderr_events_enabled=true
 
 [program:web]
-environment=NEW_RELIC_CONFIG_FILE=conf/app.ini
+environment=NEW_RELIC_APP_NAME=h-web
```

Comment:
We don't have any `[newrelic]` section in the `conf/app.ini` currently so I believe it should be fine to remove the `NEW_RELIC_CONFIG_FILE` here. We might need to re-add it in future if we want to customize the [Python agent configuration](https://docs.newrelic.com/docs/agents/python-agent/configuration/python-agent-configuration#agent-configuration-file).

Reply:
Agreed - A previous suggestion was to have a config file for each component. However, if we are only setting the application name it is a little over the top.